common-api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java

```diff
@@ -51,6 +51,7 @@ public static WorkbenchConfig createEmptyConfig() {
     public Integer clusterMaxAgeDays;
     public Integer clusterIdleMaxAgeDays;
     public String registeredDomainName;
+    public String registeredDomainGroup;
```

Comment:
It is possible, I am going to add a ticket to look at breaking our config up more as tech debt. I think its probably bigger than just this set. (And might not be worth it, because it will add more boilerplate.)

Reply:
The idea is to raise the level of abstraction. Anytime you expect two or more pieces of data to appear together, I think you really want an object.

Reply:
I'm fine with that, making a tech debt ticket.